galley/pkg/config/collection/set.go

```diff
@@ -14,6 +14,11 @@
 
 package collection
 
+import (
+	"sort"
+	"strings"
+)
+
 // Set of collections
 type Set struct {
 	collections map[Name]*Instance
```

Comment:
I know it's not from this PR, but ... It's a "Set" that contains a map? :/

Reply:
It is a set of collections. I don't mind changing names if you have strong opinions on this. It just seemed appropriate at the time.